crates/jstz_core/src/kv/transaction.rs

```diff
@@ -71,17 +79,8 @@ impl SnapshotEntry {
         }
     }
 
-    fn persistent<V>(value: V) -> Self
-    where
-        V: Value,
-    {
-        Self {
-            dirty: false,
-            value: BoxedValue::new(value),
-        }
-    }
-
     fn as_ref<V>(&self) -> &V
+    //TODO: These don't really work when it is a RefCell, since the value can be mutated without needing as_mut.
```

Comment:
Not too sure I follow this comment, `SnapshotEntry` doesn't contain any `RefCell`